log/example_new_json_logger_test.go

```diff
@@ -0,0 +1,20 @@
+package log_test
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+)
+
+// JSONLogger Example
+func ExampleNewJSONLogger() {
+	var buf bytes.Buffer
+
+	logger := log.NewJSONLogger(&buf)
+	logger.Log("question", "what is the meaning of life?", "answer", 42)
+
+	fmt.Print(&buf)
+	// Output:
+	// {"answer":42,"question":"what is the meaning of life?"}
+}
```

Comment:
@ChrisHines that's the implementation, but that behavior doesn't seem to be specified [in the package docs](http://golang.org/pkg/encoding/json). Is it declared somewhere else?